src/pages/index.tsx

```diff
@@ -27,6 +28,7 @@ export default function Component() {
   const [error, setError] = useState<string | null>(null);
   const [selectedField, setSelectedfield] = useState<string>("allEvents");
   const userEmailRef = useRef<string>("");
+  const [passwordModal, setPasswordModal] = useState(true);
```

Comment:
should be false initially @SundarakrishnanN, later on checking the /user response modify the state